packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/addconfigpkg.md

```diff
@@ -2,5 +2,7 @@ Add the latest Okta OIDC package to your project:
 
 1. Choose **File** > **Add Packages**. The Swift Package Manager opens.
 1. Enter the URL for the package in the search field:
-`https://github.com/okta/okta-oidc-ios.git`
-1. Confirm that the Dependency Rule is set to the main branch and that your project is selected, and then add the package.
+`https://github.com/okta/okta-mobile-swift.git`
+1. Confirm that the `okta-mobile-swift` package is selected, that the **Dependency Rule** is set to the main branch, and that your project is selected.
```

Comment:
We currently are stuck using the branch named `master`.  Does that matter to this?

Reply:
I'll explain why I did this and if you still think it needs changing, let me know.

I chose "main" because it reflects inclusive language. I deliberately didn't use code voice because the meaning then becomes "the main branch of the repository". I think developers understand that for most repositories that's still "master", but it also means that when we can finally change it, no documentation change is required.